code/modules/admin/randomverbs.dm

```diff
@@ -1107,7 +1107,7 @@
 		else
 			src.preview_icon = new /icon('icons/mob/human.dmi', "body_[g]")
 
-		if(!(src.mutantrace && src.mutantrace.override_skintone))
+		if(!(src.mutantrace /* && src.mutantrace.override_skintone */))
```

Comment:
I honestly don't know what's going on there either. I might've commented that out early on because it was causing runtimes or something. I'll try uncommenting them and seeing what happens.